bi_view_editor/README.rst

```diff
@@ -0,0 +1,98 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==============
+BI View Editor
+==============
+
+The module BI View Editor is user-friendly tool (Query Builder) integrated in Odoo.
+It creates views for Odoo 8, allowing the user to specify graphically the sources
+and the attributes of the data-set to analyze, automating all the operations for
+creating all the necessary low level Odoo objects.
+
+
+It is intended for:
+
+ - Graphically define your analysis data-set
+ - Add new computed information to your data set
+ - Specify the visualization attributes
+ - Create the basis for a new data exploration method in Odoo
+
+
+Benefits would be:
+
+ - More freedom to consultants or end users to customize views
+ - No more external tools for data analysis
+ - Easy update of your BI reports
+ - No more external tools for data analysis
+
+
+
+Usage
+=====
+
+
+To graphically design your analysis data-set:
+
+ - From the Reporting menu, select "Custom BI Views"
+ - Browse trough the business objects in the Query tab
+ - Pick the interesting fields (Drag & Drop)
+ - Save and click "Generate BI View"
```

Comment:
@astirpe Please remove the space before the dash to make those lists, otherwise it adds an extra indentation.